<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMoney</class>
 <widget class="QDialog" name="AddMoney">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>20</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="MoneyNum">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>50000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Confirm">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Tips_Label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>请输入1-50000范围内的整数值！</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
